src/Marten.Testing/performance_tuning.cs

```diff
@@ -22,6 +22,11 @@ public class JilSerializer : ISerializer
         private readonly Options _options 
             = new Options(dateFormat: DateTimeFormat.ISO8601, includeInherited:true);
 
+        public void ToJson(object document, TextWriter writer)
```

Comment:
@jeremydmiller Can we change the seam? From semver perspective this is going to be a breaking change (major). We could provide a new interface that would be implemented by serializers. For now I assume it's ok to change this.